packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -1374,6 +1408,27 @@ export function _useFakeChatAdapters(args: _FakeChatAdapterArgs): _FakeChatAdapt
 // @beta
 export const useTeamsCallAdapter: (args: Partial<TeamsCallAdapterArgs>, afterCreate?: ((adapter: TeamsCallAdapter) => Promise<TeamsCallAdapter>) | undefined, beforeDispose?: ((adapter: TeamsCallAdapter) => Promise<void>) | undefined) => TeamsCallAdapter | undefined;
 
+// @public
+export interface VideoBackgroundBlurEffect {
+    // (undocumented)
+    effectType: 'Blur';
+}
+
+// @beta
+export interface VideoBackgroundEffectsOptions {
+    backgroundImages?: BackgroundImage[];
+    backgroundUploadHandler?: BackgroundUploadHandler;
+    disableVideoEffects?: boolean;
```

Comment:
Open note as we can't answer until we get finialized UX designs.
We can't fully disable the video effects at the composite layer, we can only hide the UI buttons.
Depending on the final UX this should likely live on the CallControlsOptions interface
```tsx
<CallComposite options={{ callControls: { videoEffects: false } }} />
```